pkg/Microsoft.Private.PackageBaseline/baseline.packages.props

```diff
@@ -284,7 +284,7 @@
       <Version>4.0.2</Version>
     </BaseLinePackage>
     <BaseLinePackage Include="System.Runtime.Serialization.Primitives">
-      <Version>4.1.1</Version>
+      <Version>4.2.0</Version>
```

Comment:
Assuming we do go through with this change, today all contract versions need to be updated in TFS otherwise you will break those builds.